### PR TITLE
Add Flags Button to `CN` Version

### PR DESCRIPTION
**Pull request by storopoli**
This PR adds the flags to `CN` `index.md`.

### CODE REVIEW

File: contents/index.md [hunk 1]
```diff
@@ -1,5 +1,25 @@
...
+
+.language-switcher button {
+    font-size: 20px;
+}
+</style>
+
+<div class="language-switcher">
+<a href="https://juliadatascience.io"><button>🇺🇸</button></a>
+<a href="https://juliadatascience.io/pt"><button>🇧🇷</button></a>
+<a href="/"><button>🇨🇳</botton></a>
```
Comment (rikhuijzer) [inline contents/index.md]:
```suggestion
<a href="https://cn.julialang.org/JuliaDataScience/"><button>🇨🇳</botton></a>
```

**Comment by storopoli** [comment]
Tagging @findmyway and  @guixinliu because neither @rikhuijzer nor me have merge privileges

File: contents/index.md [hunk 2]
```diff
@@ -1,5 +1,25 @@
...
+.language-switcher button {
+    font-size: 20px;
+}
+</style>
+
+<div class="language-switcher">
+<a href="https://juliadatascience.io"><button>🇺🇸</button></a>
+<a href="https://juliadatascience.io/pt"><button>🇧🇷</button></a>
+<a href="https://cn.julialang.org/JuliaDataScience/"><button>🇨🇳</botton></a>
+</di
```
Comment (guixinliu) [inline contents/index.md:21]:
````suggestion
</div>
```
````

Comment (guixinliu) [inline contents/index.md]:
````suggestion
</div>
```
````

**Review by guixinliu — CHANGES_REQUESTED** [review]
The `<div>` tag at 17th line and three quotes  ` ``` ` at  3rd line should be closed.

**Comment by guixinliu** [comment]
Thank you @rikhuijzer  and @storopoli !

**Comment by storopoli** [comment]
Thank you!

**Pull request merged**